app/create_file.py

```diff
@@ -1 +1,46 @@
-# write your code here
+import os
+import sys
+import datetime
+
+
+def create_directory(new_directories: list) -> str:
+    path_to_new_directory = os.path.join(*new_directories)
+    if not os.path.exists(path_to_new_directory):
+        os.makedirs(path_to_new_directory)
+    return path_to_new_directory
+
+
+def create_file(new_file_name: str) -> None:
+    with open(new_file_name, "a") as new_file:
+        current_date_time = datetime.datetime.now()
+        timestamp = current_date_time.strftime("%Y-%m-%d %H:%M:%S")
+        new_file.write(timestamp + "\n")
+        line_number = 1
+        while True:
+            content = input("Enter content line: ") + "\n"
+            if content == "stop\n":
+                break
+            new_file.write(f"{line_number} {content}")
+            line_number += 1
```

Comment:
"\n" has to be rewritten.
```suggestion
            content = input("Enter content line: ")
            if content == "stop":
                break
            new_file.write(f"{line_number} {content}\n")
            line_number += 1
```